crates/nargo/tests/test_data/merkle_insert/Prover.toml

```diff
@@ -0,0 +1,22 @@
+old_root = "0x0620374242254671503abf57d13969d41bbae97e59fa97cd7777cd683beb9eb8"
+old_leaf = "0x1cdcf02431ba623767fe389337d011df1048dcc24b98ed81cec97627bab454a0"
+old_hash_path = [
+    # "0x1cdcf02431ba623767fe389337d011df1048dcc24b98ed81cec97627bab454a0",
```

Comment:
Can you remove the un-needed highlights?

Reply:
Done